frontend/src/utils/validation.js

```diff
@@ -78,6 +113,39 @@ class Validation {
     return { errorMessages: errorMessages, valid: valid };
   }
 
+  locationValidation(location) {
+    let keys = Object.keys(location);
+    let errorMessages = keys.reduce((accumulator, value) => {
+      return { ...accumulator, [value]: "" };
+    }, {});
+    let valid = true;
+    for (var i = 0; i < keys.length; i++) {
+      switch (keys[i]) {
+        case "address":
+          errorMessages.address = this.validateAddress(location.address);
+          break;
+        case "city":
+          errorMessages.city = this.validateCity(location.city);
+          break;
+        case "zipCode":
+          errorMessages.zipCode = this.validateZipCode(location.zipCode);
+          break;
+        case "countryId":
+          errorMessages.country = this.validateCountry(location.countryId);
+          break;
+        default:
```

Comment:
Please add this to default case:
errorMessages.country = this.validateCountry(location.countryId);
          break;